popup.js

```diff
@@ -18,7 +18,7 @@ chrome.runtime.onMessage.addListener(function(request) {
   if (request.type === "LOG_WORK_IN_JIRA_DE") {
     chrome.tabs.query({ active: true, currentWindow: true }, function(tabs) {
       chrome.tabs.executeScript(tabs[0].id, {
-        code: `(${logWork}('${request.payload.timeSpent}', '${request.payload.taskId}', '${request.payload.taskName}', '${request.payload.taskDesc}', 'de'))`
+        code: `(${logWork}('${request.payload.timeSpent}', '${request.payload.taskId}', '${request.payload.taskName}', '${request.payload.taskDesc}', 'de', '${request.payload.jiraPLTaskID}'))`
```

Comment:
if it is a string why do YOu need a template literal?

Reply:
because this string contain JS references :)